src/PrestaShopBundle/Resources/config/services/core/grid/grid_data_factory.yml

```diff
@@ -180,3 +180,20 @@ services:
         - '@prestashop.core.hook.dispatcher'
         - '@prestashop.core.grid.query.doctrine_query_parser'
         - 'cms_page'
+
+    prestashop.core.grid.data.factory.order:
+      class: '%prestashop.core.grid.data.factory.doctrine_grid_data_factory%'
+      arguments:
+        - '@prestashop.core.grid.query_builder.order'
+        - '@prestashop.core.hook.dispatcher'
+        - '@prestashop.core.grid.query.doctrine_query_parser'
+        - '!php/const PrestaShop\PrestaShop\Core\Grid\Definition\Factory\OrderGridDefinitionFactory::GRID_ID'
```

Comment:
We can stick to `order` for now but use constants all at once in a global PR

Reply:
The PR could also provide constants for https://github.com/PrestaShop/PrestaShop/pull/13988/files#diff-5c353273f197707b8ac513ef4dbb4b2cR51

Reply:
You mean have another PR that replaces hardcoded strings with constants everywhere?